doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,6 +1,22 @@
 # Upgrade Notes
 
 ## 10.5.8
+- [Twig] Sending mails and Dataobject Text Layouts, which allow rendering user controlled twig templates are now executed in a sandbox with restrictive security policies for tags, filters, functions and son on.
+         Please use following configuration to allow more tags, filters, properties, methods in template rendering:
+  ```yaml
+  pimcore:
+        templating_engine:
+            twig:
+              sandbox_security_policy:
+                tags: ['if']
+                filters: ['upper']
+                methods:
+                    Article: ['getTitle', 'getBody']
+                properties:
+                    Article: ['title', 'body']
```

Comment:
Do we really need that?

Reply:
yes, this is needed to whitelist the method and property calls on objects as the sandbox execution disallows everything by default.

So basically for sending emails with html body: `{{ pimcore_object(769).title }}`, I need to add the config:
```yaml
twig:
  sandbox_security_policy:
    methods:
        \Pimcore\Model\DataObject\News: ['getTitle']
    functions: ['pimcore_object']
```

Reply:
This could be a BC break for existing implementations, so i would suggest that we add a config to enable sandbox execution for email templates rendering. wdyt?

Reply:
I have re-used the Twig SecurityPolicy class to avoid performing checks for object methods and properties. Please see https://github.com/pimcore/pimcore/pull/13347/commits/aa5d1534784b99860da300390f672ff14d91299c